### PR TITLE
#308 Upgrade to qulice 0.18

### PR DESCRIPTION
**Pull request by GnusinPavel**
qulice-maven-plugin version updated to 0.18.17

### CODE REVIEW

**Comment by 0crat** [comment]
@GnusinPavel this pull request is too small, just 1 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) please review this and merge or reject

**Comment by paulodamaso** [comment]
@GnusinPavel The build is failing, please take a look

**Comment by GnusinPavel** [comment]
@paulodamaso build fixed

File: src/test/java/org/jpeek/MetricsTest.java [hunk 1]
```diff
@@ -218,13 +219,13 @@
             new Object[] {
                 "IndirectlyRelatedPairs",
                 "OCC",
-                0.6666666666666666d,
+                0.6_666_666_666_666_666d,
```
Comment (paulodamaso) [inline src/test/java/org/jpeek/MetricsTest.java]: @GnusinPavel This seems strange to me, we don't have sepparators in the decimal part of a number. I'd prefer these to stay as before, if it doesn't fail any qulice rule.

File: src/test/java/org/jpeek/MetricsTest.java [hunk 2]
```diff
@@ -218,13 +219,13 @@
...
             },
             new Object[] {"MethodMethodCalls", "OCC", 0.2d},
             new Object[] {
                 "MethodsWithDiffParamTypes",
                 "OCC",
-                0.3333333333333333d,
+                0.3_333_333_333_333_333d,
```
Comment (paulodamaso) [inline src/test/java/org/jpeek/MetricsTest.java]: @GnusinPavel This seems strange to me, we don't have sepparators in the decimal part of a number. I'd prefer these to stay as before, if it doesn't fail any qulice rule.

File: src/test/java/org/jpeek/MetricsTest.java (hunk 1 above)
Comment (GnusinPavel) [reply src/test/java/org/jpeek/MetricsTest.java]: yes, it violates qulice rule. Ok, I'll remove underscores and put suppression

File: src/test/java/org/jpeek/MetricsTest.java (hunk 2 above)
Comment (GnusinPavel) [reply src/test/java/org/jpeek/MetricsTest.java]: yes, it violates qulice rule. Ok, I'll remove underscores and put suppression

**Comment by GnusinPavel** [comment]
@paulodamaso fixed, please go on

**Comment by GnusinPavel** [comment]
@paulodamaso ping )

**Comment by paulodamaso** [comment]
@GnusinPavel Thanks

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@GnusinPavel Would you please correct the puzzle? Seems that it is violating @0pdd rules

**Comment by GnusinPavel** [comment]
@paulodamaso done

**Comment by paulodamaso** [comment]
@GnusinPavel The word count is all right, but the puzzle is still not ok: all the lines but the first must start with a blank space, or @0pdd won't recognize it as a puzzle again. If you have any doubt, take a look at the other multi-lined puzzles in the codebase and use one of them as example

**Comment by GnusinPavel** [comment]
@paulodamaso once again

**Comment by paulodamaso** [comment]
@GnusinPavel Thank you.

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:yegor256/jpeek#351` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Pull request merged**